packages/generator/src/generator.js

```diff
@@ -47,6 +54,8 @@ export default class Generator {
     // Wait for nuxt be ready
     await this.nuxt.ready()
 
+    process.EXPORT_HASH = createTimeHash(new Date().toISOString())
```

Comment:
This util does not accept an argument (time). Also what do you think to first attempt reading from builder instance to have a consistent hash?

Reply:
You mean setting the export hash equal to the build hash and only have it differently on `nuxt generate --no-build`?